src/components/expandable/expandable.tsx

```diff
@@ -73,11 +74,15 @@ const ExpandableText = styled.div`
   margin-top: 12px;
 `;
 
-export const Expandable = ({ summary, children }: ExpandableProps) => {
+export const Expandable = ({
+  summary,
+  children,
+  className,
+}: ExpandableProps) => {
   const [isOpen, setIsOpen] = useState(false);
 
   return (
-    <ExpandableContainer>
+    <ExpandableContainer className={className}>
```

Comment:
While tinkering around with the service content I had expandables which I wanted to update. I removed them but the changes are left and make sense so this component can be styled in the future.